packages/app-bridge/src/AppBridgePlatformApp.spec.ts

```diff
@@ -62,10 +64,22 @@ describe('AppBridgePlatformApp', () => {
         await expect(() => platformApp.api({ name: 'getCurrentUser' })).rejects.toThrow();
     });
 
-    it('should return undefined state as it is not implemented', async () => {
+    it('should return empty state when not inititalized', async () => {
         const platformApp = new AppBridgePlatformApp();
-        const state = platformApp.state();
-        expect(state).toEqual(undefined);
+        const state = platformApp.state().get();
+        expect(state).toEqual({ settings: {} });
+    });
+
+    it('should return state after app is initialized', async () => {
+        window.location.search = `?token=${TOKEN}`;
+        const platformApp = new AppBridgePlatformApp();
+        platformApp.subscribe('Context.connected', (connected) => {
+            const state = platformApp.state().get();
+
+            expect(connected).toBe(true);
+            expect(state).toEqual({ settings: 'settings-test' });
+        });
+        platformApp.dispatch(openConnection());
```

Comment:
This probably has no effect on the test, and can be removed?

Reply:
Should throw an error when the setting object is not correct, did they not do anything for you?

Reply:
No errors for me 🤷 The expectations have already run, so it shouldn't have any effect

Reply:
![Screenshot 2023-10-27 at 10 43 55](https://github.com/Frontify/brand-sdk/assets/14964896/6f9cea48-8ea9-45ad-af46-4fedb2b5c431)

For me it throws an assertionError, hm

Reply:
what is the asdf-ending settings value?

For me it passes nicely when I run it on terminal 🤷 
<img width="818" alt="image" src="https://github.com/Frontify/brand-sdk/assets/90334386/b3865faf-e301-41aa-b169-7e14b107a140">

Reply:
Ahhh i see, if you dont put the dispatch then the subscribe doesnt trigger. Its a pub sub pattern, and the dispatch initiates the "publishing" here. Only when the dispatch publishes something then the subscribe is triggered.

Reply:
Ah, I see. Would it then make sense to move the dispatch before the subscribe?

Reply:
First you need to subscribe to smth, then you can dispatch a method on it.

Reply:
If you put the dispatch in front of the subscribe, then the test doesnt fail because the subscribe is not executed, i would assume, but didnt test it